src/visualizations/VisTimeline.vue

```diff
@@ -138,6 +138,23 @@ export default {
         const end =
           (this.queriedInterval && this.queriedInterval[1]) ||
           _.max(_.map(items, item => item.end));
+        const filterDuration = (document.getElementById('duration') as HTMLInputElement).value;
```

Comment:
You can get the duration with the `queriedInterval` as show by this
```
return [moment().subtract(this.duration, 'seconds'), moment()]
```

That means you can substract `this.queriedInterval[1].substract(this.queriedInterval[0]` to get the duration in seconds.